backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/database/repos/CardRepository.kt

```diff
@@ -75,36 +73,48 @@ object CardRepository {
         }
     }
 
-    fun getCardStatisticsByProjectAndRegion(project: String, dateStart: Timestamp, dateEnd: Timestamp, regionId: Int?): List<CardStatisticsResultModel> {
+    fun getCardStatisticsByProjectAndRegion(
```

Comment:
Why the changing of the naming? I think "dateStart" and "dateEnd" are somehow clearer than "from" and "until".

Reply:
I think date sounds like a date (as in a local date or a a day of the calendar). However they are instants in time. We can also revert it though...